index.html

```diff
@@ -113,13 +113,13 @@
     //
     //-------------------------------------------------------------------------
 
```

Comment:
identifying what the variables are used for would be helpful in looking at the code.